package.json

```diff
@@ -2,7 +2,7 @@
   "main": "lib/lib.js",
   "typings": "lib/kayenta/index.d.ts",
   "name": "@spinnaker/kayenta",
-  "version": "0.0.73",
+  "version": "0.0.75",
```

Comment:
im not sure why this is showing .73 => .75. In master this is showing .74: https://github.com/spinnaker/deck-kayenta/blob/master/package.json#L5

Reply:
If you rebase this against latest master it should be `0.0.74` as expected